codestyle/checkstyle.xml

```diff
@@ -204,6 +204,12 @@
       <property name="message" value="Use org.apache.druid.server.http.HttpMediaType#TEXT_PLAIN_UTF8 instead"/>
     </module>
 
+    <module name="Regexp">
+      <property name="format" value='^((?!throw|=).)* Throwables.propagate\('/>
```

Comment:
`=` doesn't work. The assignment will simply fail with an exception. There is a bug in `HadoopConverterJob`, line 347.

Reply:
checkstyle plugin fails when I don't add `|=` but passes when I add it. I can also correctly search for occurrences via Regex search at Intellij IDEA Find dialog. Which exception do you get?

Reply:
You mean that there is an error inside the plugin, like it breaks up, or that there are violations reported by the plugin? If the latter, those violations should be fixed, because they are bugs.

Reply:
I mean that both IntelliJ IDEA search panel and checkstyle plugin works as expected when I add `|=`. I don't get an error at `HadoopConverterJob`.

Reply:
What happens when you remove `|=`?